bindings/python/src/session.cpp

```diff
@@ -465,6 +466,28 @@ namespace
        return ret;
     }
 
+    list cached_piece_info_list(std::vector<cached_piece_info> const& v)
+    {
+       list pieces;
+       ptime now = time_now();
```

Comment:
I believe this is the problem. the ``ptime`` typedef is deprecated, you should use ``lt::time_point`` instead.